include/quda_internal.h

```diff
@@ -44,6 +44,11 @@
 #define USE_TEXTURE_OBJECTS
 #endif
 
+// if not using texture objects then we need to disable multi-blas support since these don't work with texture references
+#ifndef USE_TEXTURE_OBJECTS
```

Comment:
should this be fixed here or in cmakelists.txt ? No real preference

Reply:
I'd say to leave it as is, since this is a short-term fix for Fermi architectures that we will remove once we branch off QUDA 0.9.